pywbem_mock/_defaultinstanceprovider.py

```diff
@@ -0,0 +1,527 @@
+            the parameter to False.
+
+          IncludeQualifiers (:class:`py:bool`):
+            Indicates that qualifiers are to be included in the returned
+            instance, as follows:
+
+            * If `False`, qualifiers are not included.
+            * If `True`, qualifiers are included if the WBEM server implements
+              support for this parameter.
+            * If `None`, the :term:`DSP0200` defined server default `False`
+              is used.
+
+            This parameter has been deprecated in :term:`DSP0200`. Clients
+            cannot rely on qualifiers to be returned in this operation.
+
+          IncludeClassOrigin (:class:`py:bool`):
+            Indicates that class origin information is to be included on each
+            property in the returned instance, as follows:
+
+            * If `False`, class origin information is not included.
+            * If `True`, class origin information is included.
+            * If `None`, uses :term:`DSP0200` server default `False`.
+
+          PropertyList (:term:`string` or :term:`py:iterable` of :term:`string`):
+            An iterable specifying the names of the properties (or a string
+            that defines a single property) to be included in the returned
+            instance (case independent).
+
+            An empty iterable indicates to include no properties.
+
+            If `None`, all properties are included.
+
+        Returns:
+
+            A :class:`~pywbem.CIMInstance` object that is a representation of
+            the retrieved instance.
+            Its `path` attribute is a :class:`~pywbem.CIMInstanceName` object
+            with its attributes set as follows:
+
+            * `classname`: Name of the creation class of the instance.
+            * `keybindings`: Keybindings of the instance.
+            * `namespace`: Name of the CIM namespace containing the instance.
+            * `host`: `None`, indicating the WBEM server is unspecified.
+
+        Raises:
+
+            CIMError:  CIM_ERR_INVALID_NAMESPACE, CIM_ERR_INVALID_PARAMETER
+              CIM_ERR_NOT_FOUND
+        """  # noqa: E501
+        # pylint: enable=line-too-long
+
+        namespace = InstanceName.namespace
+        self.validate_namespace(namespace)
+        iname = InstanceName
+
+        # Set LocalOnly to False as fixed value.
+        LocalOnly = INSTANCE_RETRIEVE_LOCAL_ONLY
+
+        instance_store = self.get_instance_store(namespace)
+
+        if not self._class_exists(namespace, iname.classname):
+            raise CIMError(
+                CIM_ERR_INVALID_CLASS,
+                _format("Class {0!A} for GetInstance of instance {1!A} "
+                        "does not exist.", iname.classname, iname))
+
+        return self._get_instance(iname, instance_store,
+                                  LocalOnly,
+                                  IncludeClassOrigin,
+                                  IncludeQualifiers, PropertyList)
```

Comment:
This method is missing in the class hierarchy in the current code.

Reply:
Fixed.  DONE